packages/client/src/runtime/core/engines/data-proxy/utils/getClientVersion.ts

```diff
@@ -18,7 +18,7 @@ async function _getClientVersion(host: string, config: EngineConfig) {
   }
 
   // for data proxy v2, or accelerate, resolution isn't needed
-  if (host.includes('accelerate') && clientVersion !== '0.0.0') {
+  if (host.includes('accelerate') && clientVersion !== '0.0.0' && clientVersion !== 'in-memory') {
```

Comment:
We also handle `in-memory` now so you can actually test studio locally. I think this should not be needed anymore once I merge https://github.com/prisma/studio-code/pull/514